docs/advanced_topics/images/renditions.md

```diff
@@ -36,6 +36,24 @@ See also: [](image_tag)
 
 (prefetching_image_renditions)=
 
+## Generating multiple renditions for an image
+
+You can generate multiple renditions of the same image from Python using the native `get_renditions()` method. It will accept any number of 'specification' strings, and will generate a set of matching renditions much more efficiently than generating them one-by-one. For example:
```

Comment:
Suggestion: Might be worth noting here that renditions are still generated sequentially

Reply:
Reworded in [176aa23](https://github.com/wagtail/wagtail/pull/9550/commits/176aa237992beb7d2c02cc66f7ff69e517618775)